auslib/web/common/releases_spec.yml

```diff
@@ -0,0 +1,129 @@
+        - name: names_only
+          description: If "names_only" is set to true, the response changes format and provides a list of all of the Release names.
+          in: query
+          required: false
+          type: string
+      responses:
+        200:
+          description: A releases list or a list of releases names.
+          schema:
+            $ref: '#/definitions/ReleasesWrapper'
+
+  /api/v1/releases/{release}:
+    get:
+      operationId: 'auslib.web.common.releases.get_release'
+      description: Returns the "data" portion of the named Release, which is a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: pretty
+          description: If "pretty" is present in the query string and set to true, it will be pretty formatted.
+          in: query
+          required: false
+          type: integer
+      responses:
+        200:
+          description: Release Blob
+
+  /api/v1/releases/{release}/revisions:
+    get:
+      operationId: 'auslib.web.common.releases.get_release_history'
+      description: Returns previous versions of the named Release in a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: page
+          in: query
+          required: false
+          type: integer
+          format: int32
+        - name: limit
+          in: query
+          required: false
+          type: integer
+          format: int32
+      responses:
+        200:
+          description: Release History.
+        400:
+          description: Bad input.
+
+  /api/v1/releases/{release}/builds/{platform}/{locale}:
+    get:
+      operationId: 'auslib.web.common.releases.get_release_single_locale'
+      description: Returns the platform+locale specific data of the named Release, which is a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: platform
+          description: Platform
+          in: path
+          required: true
+          type: string
+        - $ref: '#/parameters/localeParam'
+      responses:
+        200:
+          description: Release Blob.
+          headers:
+            X-Data-Version:
+              description: The version of returned data.
+              type: integer
+        404:
+          description: Release not found for platform+locale.
+
+definitions:
+  Release:
+    type: object
+    properties:
+      name:
+        type: string
+      product:
+        type: string
+      read_only:
+        type: ['boolean', 'null']
+      rule_ids:
+        type: array
+        items:
+          type: integer
+
+  ReleasesWrapper:
+    type: object
+    properties:
+      names:
+        type: array
+        items:
+          type: string
+      releases:
+        type: array
+        items:
+          $ref: '#/definitions/Release'
```

Comment:
Is there a reason that the "Release" definition exists when this is the only reference to it?